README.md

```diff
@@ -1 +1,37 @@
-# megafuel-js-sdk
\ No newline at end of file
+# megafuel-js-sdk
+
+This is Javascript SDK for the MegaFuel clients, it provides a simple way to interact with the MegaFuel.
+
+For more information, please refer to the [API documentation](https://docs.nodereal.io/docs/megafuel-api).
+
+## Quick Start
+1. Install dependency
+ ```shell
+ $ npm install
+ ```
+
+2. Example
+```js
+import {ethers} from "ethers";
+import {SponsorClient, WhitelistType} from 'megafuel-js-sdk';
+
+const SPONSOR_URL = "https://sponsor-api.nodereal.io";
```

Comment:
This seems a very wrong endpoint even without an API key.

Reply:
By the way, because we provide fixed endpoint for testnet and mainnet, can we define some const so that user dont have to input the URL by themselves?
Of cource, we can still allow develop to custom the endpoint

Reply:
the fixed ep is paymaster URL, not fixed for sponsor URL, I think no need to do that. just provide them in examples.